activerecord/test/cases/helper.rb

```diff
@@ -21,6 +21,9 @@
 # Show backtraces for deprecated behavior for quicker cleanup.
 ActiveSupport::Deprecation.debug = true
 
+# Don't add indices to nested attribute errors
+ActiveRecord::Base.index_nested_attribute_errors = false
```

Comment:
This should be the default value already, so we should not need to set it on the test.